src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -281,7 +281,9 @@ export default class ReviewCollapsibleChapter extends React.Component {
                 aria-controls={`collapsible-${this.id}`}
                 onClick={this.props.toggleButtonClicked}
               >
-                {chapterTitle || ''}
+                {chapterTitle && (
+                  <h3 className="vads-u-font-size--h4">{chapterTitle}</h3>
```

Comment:
@Mottie I have one ask, and I hope it's a small one. Can we reorder this structure so it's H3 > button ? Axe agrees this is a good change, W3C html validator says it would be better the other way around. I ran a quick snippet through to confirm before requesting the change. Screenshot attached below.

LMK if this is out of scope. I'm happy to add a follow on ticket as needed.

---
![Screen Shot 2021-01-21 at 10 46 54 AM](https://user-images.githubusercontent.com/934879/105389169-2618ed80-5bdd-11eb-9f34-c2fe061eb6ab.png)

Reply:
It's not a problem! I only put the `h3` inside because of the margin it adds to the accordion

![](https://user-images.githubusercontent.com/136959/103824535-bf6ece00-5039-11eb-9e12-105b58f34f37.png)

See the "Using an h4 wrapper" example in the screenshot. There is a big gap between the accordion above it. It should be easy to resolve by adding a class to remove the margin.

Reply:
Done